docs/faq.md

```diff
@@ -153,6 +154,10 @@ See [Extended Usage](../jib-gradle-plugin#extended-usage) for the `container.for
 
 Normally, the plugin sets a default entrypoint for java applications, or lets you configure a custom entrypoint using the `container.entrypoint` configuration parameter. You can also override the default/configured entrypoint by defining a custom entrypoint when running the container. See [`docker run --entrypoint` reference](https://docs.docker.com/engine/reference/run/#entrypoint-default-command-to-execute-at-runtime) for running the image with Docker and overriding the entrypoint command, or see [Define a Command and Arguments for a Container](https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/) for running the image in a [Kubernetes](https://kubernetes.io/) Pod and overriding the entrypoint command.
 
+### I want to containerize an executable JAR.
+
+Although Jib supports the ability to add an executable JAR to the container and run it using a custom `java -jar` entrypoint command, the intention of Jib is to add individual class files and dependency JARs into the container instead of putting a runnable JAR into the container. This lets Jib choose an opinionated, optimal layout for the application on the container image, which also allows it to skip the extra JAR-packaging step.
```

Comment:
How about "Although it is possible to configure Jib to add an ..."? I think it will not be so trivial to make that work, because you need to to first package a JAR, copy the JAR into an isolated location, which may need some time to research (you don't want to have the entire `target` directory as `<extraDirectory>`), and configure `<extraDirectory>` and `<entrypoint>`.

Reply:
And basically the same argument applies to a non-executable JAR (in which case you'd have to do `java -cp ...`). Can we somehow mention this too?

Oh, and this sort of applies to WAR too (we put exploded WAR contents), but I think I don't want to mention WAR here for now, because although many popular Servlet engines support placing an exploded WAR for deployment, I wonder if there can be some server that doesn't support it. We will watch that.